ddtrace/contrib/flask/patch.py

```diff
@@ -71,7 +72,10 @@
 
 def get_version():
     # type: () -> str
-    return getattr(flask, "__version__", "")
+    try:
+        return importlib_metadata.version("flask")
```

Comment:
🚲 🏠 (closest to "bike shed" I can do).

Should we offer an API in `ddtrace.internal.packages` for getting a single package's version? Then we could also cache it like the others.


Definitely not urgent to address, just a thought.

Reply:
Yes, it would be great!